pom.xml

```diff
@@ -44,6 +44,19 @@
           <target>1.8</target>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+              <mainClass>fully.qualified.MainClass</mainClass>
+            </manifest>
+          </archive>
+          <descriptorRefs>
+           <descriptorRef>jar-with-dependencies</descriptorRef>
+          </descriptorRefs>
+        </configuration>
+      </plugin>
```

Comment:
You shouldn't have to need the assembly plugin?

Reply:
Since I put the nsq client to the es loader's repo for now, I used assembly plugin but I will remove it while I am adding CI/CD to project